TLM/.editorconfig

```diff
@@ -2,11 +2,9 @@ root=true
 
```

Comment:
Did you intend to include `.editorconfig` in the PR?

Reply:
yes. It was driving me crazy!